bigchaindb/common/transaction.py

```diff
@@ -229,6 +229,8 @@ def __init__(self, fulfillment, public_keys=None, amount=1):
             raise TypeError('`amount` must be an int')
         if amount < 1:
             raise AmountError('`amount` must be greater than 0')
+        if amount > 9 * 10 ** 18:
```

Comment:
Can we make this a const somewhere so that we don't need to recalculate this value everytime?